ironfish/src/network/messages/newTransactionV2.test.ts

```diff
@@ -1,17 +1,38 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import { randomBytes, randomInt } from 'crypto'
+import { createNodeTest, useTxSpendsFixture } from '../../testUtilities'
 import { NewTransactionV2Message } from './newTransactionV2'
 
 describe('NewTransactionV2Message', () => {
-  it('serializes the object into a buffer and deserializes to the original object', () => {
-    const transactions = [...new Array(10)].map((_) => randomBytes(randomInt(500, 10000)))
+  const nodeTest = createNodeTest()
+
+  function expectNewTransactionV2MessageToMatch(
+    a: NewTransactionV2Message,
+    b: NewTransactionV2Message,
+  ): void {
+    // Test transactions separately because Transaction is not a primitive type
+    expect(a.transactions.length).toEqual(b.transactions.length)
+    a.transactions.forEach((transactionA, transactionIndexA) => {
+      const transactionB = b.transactions[transactionIndexA]
+
+      expect(transactionA.hash().equals(transactionB.hash())).toBe(true)
+    })
+
+    expect({ ...a, transactions: undefined }).toMatchObject({ ...b, transactions: undefined })
+  }
+
+  // eslint-disable-next-line jest/expect-expect
```

Comment:
Same comment here re: jest expects.

Reply:
Updating linting to allow for functions that start with expect doesn't allow for function names that include digits. I tried playing around with the glob pattern but wasn't able to find a solution so I've manually disabled `jest/expect-expect` here and for `newBlockV2.test.ts`

Reply:
Can you remove the `eslint-disable` here?

Reply:
Sounds good -- Looks like something they've fixed in a newer version of eslint-plugin-jest